pages/create-account.js

```diff
@@ -0,0 +1,143 @@
+import Head from 'next/head';
+import React, { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Account() {
+  const [errors, setErrors] = useState({
+    fname: '',
+    lname: '',
+    uname: '',
+    pword: '',
+    confirmpword: '',
```

Comment:
For these variables write out the entire name they aren't that much longer and will be easier to look back at instead of guessing what the variables represent.

```suggestion
    firstName: '',
    lastName: '',
    username: '',
    password: '',
    confirmPassword: '',
```